tests/test_decorators.py

```diff
@@ -11,7 +11,10 @@
 
 import unittest
 
-from django.core.urlresolvers import reverse
+try:
+    from django.urls import reverse
```

Comment:
Can you add comments here for which version of Django this and the fallback are for? Will make cleaning up when support for older versions is dropped easier.